### PR TITLE
Adding validator details 'Avatar', 'Validator Info' and 'Statistics' sections

### PR DESCRIPTION
**Pull request by stanfilip0v**
Now displaying the actual Validator details data on the 'Avatar', 'Validator Info' and 'Statistic' sections instead of filling it with dummy data

CUDOS-886, CUDOS-887

### CODE REVIEW

**Review by SpaghettiOverload — APPROVED** [review]

File: src/containers/ValidatorDetails/components/Details/components/ValidatorInfo/utils.ts [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    return results
+  }
+
+  if (status === 3) {
+    results.status = 'active'
+  } else if (status === 2) {
+    results.status = 'unbonding'
+  } else if (status === 1) {
+    results.status = 'unbonded'
+  } else {
```
Comment (maptuhec) [inline src/containers/ValidatorDetails/components/Details/components/ValidatorInfo/utils.ts]: can we skip all of the `else if` structure and just use `if (status=== sth) {
 se status} it is way more clearer and easier to maintain

File: src/containers/ValidatorDetails/components/Details/components/ValidatorInfo/utils.ts [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+  return results
+}
+
+export const getCondition = (condition: number, status: number) => {
+  let result = 'n/a'
+  if (status === 3) {
+    if (condition > 90) {
+      result = 'good'
+    } else if (condition > 70 && condition < 90) {
```
Comment (maptuhec) [inline src/containers/ValidatorDetails/components/Details/components/ValidatorInfo/utils.ts]: same here

**Review by maptuhec — APPROVED** [review]

File: src/containers/ValidatorDetails/components/Details/components/ValidatorInfo/utils.ts (hunk 1 above)
Comment (stanfilip0v) [reply src/containers/ValidatorDetails/components/Details/components/ValidatorInfo/utils.ts]: resolved

File: src/containers/ValidatorDetails/components/Details/components/ValidatorInfo/utils.ts (hunk 2 above)
Comment (stanfilip0v) [reply src/containers/ValidatorDetails/components/Details/components/ValidatorInfo/utils.ts]: resolved

**Pull request merged**